src/wasm/types.rs

```diff
@@ -119,6 +119,13 @@ type CardanoCertificate =
         keypath: Keypath
         poolKeyhash: Uint8Array
       }
+    }
+  | {
+      voteDelegation: {
+        keypath: Keypath
+        type: number
+        drepCredhash: Uint8Array | undefined | null
```

Comment:
why undefined or null? Seems like `drepCredhash?: Uint8Array` should be enough.

in scripts/build-protos.rs, you could rename this to `drepCredHash` (proper camelCase)